packages/server/test/adapters/awsLambda.test.tsx

```diff
@@ -181,6 +189,96 @@ Object {
 `);
 });
 
+test('test v2 format with multiple / in query key', async () => {
```

Comment:
I have added a test case to make sure I don't break the multiple / in query key case with this change